src/main/resources/liquibase/crop_changelog/13_2_0.xml

```diff
@@ -486,5 +486,74 @@
 		</sql>
 	</changeSet>
 
+	<changeSet author="vanina" id="v13.2.0-1">
+		<preConditions onFail="MARK_RAN">
+			<not>
```

Comment:
@vmaletta we could add that also exists ims_transaction.inventory_id

Reply:
Ok